hack/kind/start-kind.sh

```diff
@@ -1,18 +1,30 @@
 #!/usr/bin/env bash
+
 KIND=${KIND:-kind}
 KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-kind-grafana}
 KUBECONFIG=${KUBECONFIG:-~/.kube/kind-grafana-operator}
-CRD_NS=${CRD_NS:-grafana-crds}
-
+RECREATE_CLUSTER=${RECREATE_CLUSTER:-yes}
```

Comment:
What's the use-case for cluster re-creation?

Reply:
It was the previous behaviour, I just retained it.
Since you're not using the functionality I would nuch rather just re-use the a cluster and get rid of the recreate stuff

Reply:
Ah, I see, you just moved it under a flag. Alright, thx for clarification :)